docs/paper/dev/getting-started/manual-project-setup.md

```diff
@@ -1,9 +1,9 @@
 ---
-slug: /dev/project-setup
+slug: /dev/manual-project-setup
 description: Step-by-step instructions on how to setup a plugin development environment.
 ---
 
-# Paper Project Setup
+# Manual Paper Project Setup
```

Comment:
Bleh, don’t like manual. Just leave it as project setup and then have the mcdev example

Reply:
alright